feature-libs/asm/tsconfig.lib.json

```diff
@@ -7,6 +7,7 @@
     "moduleResolution": "node",
     "declaration": true,
     "sourceMap": true,
+    "strict": true,
```

Comment:
Can you try also enabling "angularCompilerOptions.strictTemplates": true and check if it's possible to keep it?